broadway_api/__main__.py

```diff
@@ -0,0 +1,33 @@
+import logging
+
+import tornado.ioloop
+
+from broadway_api.bootstrap import (
+    initialize_logger,
+    initialize_database,
+    initialize_course_tokens,
+    initialize_signal_handler,
+    initialize_app,
+)
+
+from broadway_api import gen_global_settings, gen_flags
```

Comment:
This feels off. `gen_global_settings` and `gen_flags` are not I would expect to be exposed by a Python module at this level (e.g. would you expect `argparse` to expose its utility functions as a top-level import?).

These seem like bootstrap functions. The names `initialize_global_settings` and `initialize_flags` don't seem too far off from `gen_...` right?

Reply:
hmm yeah i should rename those.